helix-rest/src/test/java/org/apache/helix/rest/server/AbstractTestClass.java

```diff
@@ -105,7 +105,7 @@
   // For a single-ZK/Helix environment
   protected static final String ZK_ADDR = "localhost:2123";
   protected static final String WORKFLOW_PREFIX = "Workflow_";
-  protected static final String JOB_PREFIX = "Job_";
+  protected static final String JOB_PREFIX = "JOB";
```

Comment:
This change is necessary. This was not correct from the beginning of the change.